WORKSPACE

```diff
@@ -97,33 +97,3 @@ tf_workspace(tf_repo_name = "@org_tensorflow")
 load("//third_party/tf:tf_configure.bzl", "tf_configure")
 
 tf_configure(name = "local_config_tf")
-
-http_archive(
-    name = "eigen",
-    # TODO(pmassey): Probably move this content in a third_party/eigen.BUILD file
-    build_file_content = """
-cc_library(
-  name = "eigen3",
-  textual_hdrs = glob(["Eigen/**", "unsupported/**"]),
-  visibility = ["//visibility:public"],
-)
-    """,
-    sha256 = "7e7a57e33c59280a17a66e521396cd8b1a55d0676c9f807078522fda52114b5c",
-    strip_prefix = "eigen-eigen-8071cda5714d",
-    urls = [
-        "https://storage.googleapis.com/mirror.tensorflow.org/bitbucket.org/eigen/eigen/get/8071cda5714d.tar.gz",
-        "https://bitbucket.org/eigen/eigen/get/8071cda5714d.tar.gz",
-    ],
-)
```

Comment:
Wait, we didn't remove the `qsim_old` at all, so still there is a dependency on the `Eigen` in [here](https://github.com/tensorflow/quantum/blob/0c90033d8763478300a6626b038c7c7ddacb7a4a/tensorflow_quantum/core/qsim/unitary_space_test.cc#L18-L21). Shall we remove this BUILD lines after removing the entire dependent codes?